### PR TITLE
New Pipelines Update

### PR DESCRIPTION
**Pull request by cedricclyburn**

### CODE REVIEW

**Comment by btannous** [comment]
@cedricclyburn is this good to go or still a work in progress? also do you have this on your katacoda so that I can test/validate it works before merging (if so please share the link!)

Thanks!

**Comment by cedricclyburn** [comment]
@btannous Just made a small tweak to fix an update, should be good to go! [Here's](https://katacoda.com/cclyburn/courses/cicd-application-delivery/pipelines) the link to test.

File: middleware/pipelines/set-env.sh [hunk 1]
```diff
@@ -4,13 +4,12 @@ curl -LO https://letsencrypt.org/certs/lets-encrypt-x3-cross-signed.pem > /dev/n
 oc login -u developer -p developer https://master:8443 --certificate-authority=lets-encrypt-x3-cross-signed.pem --insecure-skip-tls-verify=true > /dev/null 2>&1
 
 # Get the tar.xz for tekton CLI
-curl -LO https://github.com/tektoncd/cli/releases/download/v0.13.0/tkn_0.13.0_Linux_x86_64.tar.gz
+curl -LO https://github.com/tektoncd/cli/releases/download/v0.14.0/tkn_0.14.0_Linux_x86_64.tar.gz
```
Comment (btannous) [inline middleware/pipelines/set-env.sh:7]: I'd suggest a quick update using the Openshift built releases of the tools, can find them here: https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/

**Review by btannous — COMMENTED** [review]
Looks awesome @cedricclyburn ! I will run through this and verify it all works as expected and then can merge in

File: middleware/pipelines/06trigger-pipeline.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Let's start a pipeline to build and deploy our backend application using `tkn`. By creating a `PipelineRun` with the name of our applied `Pipeline`, we can define various arguments to our command like `params` that will be used in the `Pipeline`.  For example, we can apply a request for storage with a `persistentVolumeClaim`, as well as define a name for our `deployment`, `git-url` repository to be cloned, and `IMAGE` to be created.
+
+We'll first build and deploy our backend application using the following command, with the params already included for our specific demo:
+
+`tkn pipeline start build-and-deploy \ 
+    -w name=shared-workspace,volumeClaimTemplateFile=resources/persistent_volume_claim.yaml \ 
+    -p deployment-name=vote-api \ 
+    -p git-url=https://github.com/openshift-pipelines/vote-api.git \ 
+    -p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/vote-api`{{execute}}
```
Comment (btannous) [inline middleware/pipelines/06trigger-pipeline.md:13]: This doesn't seem to work any ideas @cedricclyburn ?

**Review by btannous — CHANGES_REQUESTED** [review]

**Comment by cedricclyburn** [comment]
Fixed due to Joel's magic fix!

**Review by btannous — APPROVED** [review]

**Pull request merged**